packages/core-mobile/app/hooks/useVariableSafeAreaInsets.ts

```diff
@@ -1,15 +1,14 @@
 import { useSafeAreaInsets } from 'react-native-safe-area-context'
-import { useNetworks } from './networks/useNetworks'
+import { useSelector } from 'react-redux'
+import { selectIsDeveloperMode } from 'store/settings/advanced'
 
 export const useVariableSafeAreaInsets = (): {
   top: number
   bottom: number
   conditionalTop: number
```

Comment:
can you add a comment to document what conditionalTop is used for?

Reply:
You're right. I'll try finding it a better name as well. It can be confusing which one to take (conditional to what?).